src/ex3/ItemManager.js

```diff
@@ -0,0 +1,30 @@
+import PokemonClient from "./PokemonClient.js";
+const pokemon = new PokemonClient();
+
+export default class ItemManager {
+    constructor() {
+        this.tasks = [];
+    }
+
+    async add(item) {
+            var regex=/^[0-9,]+$/;
+            const pokemonsName = ['bulbasaur','ivysaur','charmander','charmeleon','charizard','squirtle','wartortle','blastoise','caterpie'];
+            if (item.match(regex) || pokemonsName.indexOf(item)!=-1) { //if input is nums & ',' or pokemon name from pokemonsName list
+                await (this.pokemonFetch(item));
+            }
+            else { //if input is text
+                this.tasks.push(item);
+            }
+            return this.tasks;
+    }
+
+    async pokemonFetch(item){
+        await pokemon.pokemonCatch(item)
```

Comment:
You are using both await and then which is a bad practice. You either work with async-await or with then